connectors/src/connectors/github/index.ts

```diff
@@ -703,7 +703,7 @@ export const getGithubConfig: ConnectorConfigGetter = async function (
 export async function setGithubConfig(
   connectorId: ModelId,
   configKey: string,
-  configValue: string
+  configValue: string | string[]
```

Comment:
Changing this type will actually requires to update the code below since we made assumption that it was a string. What's the need for an array of string here?

Reply:
Wanted to allow for multiple patterns. But as Stan mentioned everything can indeed be written within a single regex